pkg/network/ebpf/c/protocols/classification/maps.h

```diff
@@ -7,7 +7,7 @@
 #include "protocols/classification/structs.h"
 
 // Kernels before 4.7 do not know about per-cpu array maps.
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 7, 0)
+#if defined(COMPILE_CORE) || LINUX_VERSION_CODE >= KERNEL_VERSION(4, 7, 0)
```

Comment:
```suggestion
#if defined(COMPILE_CORE) || (COMPILE_RUNTIME && LINUX_VERSION_CODE >= KERNEL_VERSION(4, 7, 0))
```
Otherwise prebuilt will use per-cpu since it gets built on 4.7+

Reply:
This does get edited here: https://github.com/datadog/datadog-agent/blob/hasan.mahmood/co-re/pkg/network/tracer/connection/kprobe/tracer.go#L164

... but I'll make this change anyway.

Reply:
Hmm, does it make sense to have it always be per-cpu in source then? Since it will get edited if not supported?